bin/desi_proc

```diff
@@ -371,7 +371,16 @@ if rank == 0 and args.obstype == 'SCIENCE':
 
     fibermap_ok = os.path.exists(fibermap)
 
-#- if assemble_fibermap failed and obstype is SCIENCE, exit now
+    #- Some commissioning files didn't have coords* files that caused assemble_fibermap to fail
+    #- these are well known failures with no other solution, so for those, just force creation
+    #- of a fibermap with null coordinate information
+    if not fibermap_ok and int(args.night) <	20200310:
+        log.info("Since night is before 20200310, trying to force fibermap creation without coords file")
+        cmd += ' --force'
+        runcmd(cmd, inputs=[], outputs=[fibermap])
```

Comment:
Because this is still within the indented block, it will only run for data prior to 20200310.  I think you want to add `--force` if before 20200310, but then do the `if not fibermap_ok` and run for any date.

Reply:
Hi Stephen, the intention was to only have it run if before March 10th. If we want to run for future dates as well, I can change it to what you mention

Reply:
If the goal is to do this everytime if a failure occurs, then I think the correct solution is to have --force be the default

Reply:
OK, I was missing the context before this where it tries to run without --force and then here tries again with --force if the first call failed and if before 20200310.  This is fine.